src/actions/notificationPane.js

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios';
+import io from 'socket.io-client';
+export const GET_NOTIFICATIONS = 'GET_NOTIFICATIONS';
+export const GET_SOCKET_NOTIFICATIONS = 'GET_SOCKET_NOTIFICATIONS';
+export const MARK_NOTIFICATIONAS_READ = 'MARK_NOTIFICATIONAS_READ';
+export const CHANGE_NOTIFICATION_PANE_HEIGHT =
+	'CHANGE_NOTIFICATION_PANE_HEIGHT';
+
+export const GetNotifications = () => async dispatch => {
+	const Token = `Bearer ${localStorage.getItem('token')}`;
+	const headers = {
+		'Content-Type': 'application/json',
+		token: Token,
+	};
+	await axios
+		.get(`${process.env.BACKEND_BASE_URL}/api/v1/notifications`, { headers })
+		.then(res => {
+			const UserNotifications = res.data.data;
+			dispatch({
+				type: GET_NOTIFICATIONS,
+				Notifications: UserNotifications,
+			});
+		});
+};
+export const notificationPaneHeigthHandle = (
+	height,
+	viewButton,
+) => async dispatch => {
+	let notificationPaneHeight = 0;
+	let limit = 0;
+	let title = '';
+	if (height > 10 && viewButton[0]) {
+		notificationPaneHeight = 650;
+		limit = height;
+		title = 'less';
+	} else {
+		notificationPaneHeight = 500;
+		limit = 10;
+		title = 'more';
+	}
+
+	dispatch({
+		type: CHANGE_NOTIFICATION_PANE_HEIGHT,
+		notificationPaneHeight,
+		viewButton: !viewButton[0],
+		limitNumber: limit,
+		buttonTitle: title,
+	});
+};
+
+export const markNotificationAsRead = (
+	notificationID = 0,
+) => async dispatch => {
+	const Token = `Bearer ${localStorage.getItem('token')}`;
+	const headers = {
+		'Content-Type': 'application/json',
+		token: Token,
+	};
+	notificationID = notificationID === 0 ? '' : notificationID;
+	await axios
+		.patch(
+			`${process.env.BACKEND_BASE_URL}/api/v1/notifications/${notificationID}`,
+			{ isRead: true },
+			{ headers },
+		)
+		.then(res => {
+			const response = res.data.data;
+			dispatch({
+				type: MARK_NOTIFICATIONAS_READ,
+				markNotificationAsReadMessage: response,
+			});
+		});
+
+	dispatch(fetchUserNotification());
```

Comment:
@shemaeric1234 the name of this function and the action it is dispatching are kind misleading. does this "fetchUserNotification" action actually fetch user notification?